akirachix/FactFinders-Informational
# Feature/team

# Engineers Checklist

## What this PR does?
- Implemented a new "Our Team" section on the homepage, showcasing the members of your 5-person development team.
- Added profiles for each team member, including their name and role.

## How can it be tested?
- Navigate to the homepage of the application and locate the "Our Team" section. Verify that the section is present and the team member profiles are displayed.
- Test the "Our Team" section in different browsers (e.g., Chrome, Firefox, Safari, Edge) to ensure compatibility and consistent behavior.
